### ISSUE TITLE
Hard crash on Android 5.1.1 java.lang.UnsatisfiedLinkError

### ISSUE THREAD

**Issue by Thomas101** [viromedia/viro#232]
Hi,

We've just come across a hard crash on Android 5.1.1 devices. Doesn't happen on 6.0 and above. We tried in our own app first, but also see the same behaviour on the Sample App. This is on `2.5.1` and `2.5.0` of the library.

The crash happens on startup when `new ReactViroPackage()` is called in `getPackages()`, so we're not able to call `isARSupportedOnDevice` in JavaScript to check compatability. It looks like it could be the same underlying issue as #192

Here's the exception that gets thrown...

```
E/art     (31601): dlopen("/data/app/com.virosample-1/lib/arm/libviro_renderer.so", RTLD_LAZY) failed: dlopen failed: cannot locate symbol "_ZTVN10__cxxabiv117__class_type_infoE" referenced by "libLinearMath.so"...
E/AndroidRuntime(31601): FATAL EXCEPTION: Thread-1378
E/AndroidRuntime(31601): Process: com.virosample, PID: 31601
E/AndroidRuntime(31601): java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "_ZTVN10__cxxabiv117__class_type_infoE" referenced by "libLinearMath.so"...
E/AndroidRuntime(31601): at java.lang.Runtime.loadLibrary(Runtime.java:371)
E/AndroidRuntime(31601): at java.lang.System.loadLibrary(System.java:988)
E/AndroidRuntime(31601): at com.viromedia.bridge.module.MaterialManager.<clinit>(MaterialManager.java:37)
E/AndroidRuntime(31601): at com.viromedia.bridge.ReactViroPackage.createNativeModules(ReactViroPackage.java:87)
E/AndroidRuntime(31601): at com.facebook.react.NativeModuleRegistryBuilder.processPackage(NativeModuleRegistryBuilder.java:106)
E/AndroidRuntime(31601): at com.facebook.react.ReactInstanceManager.processPackage(ReactInstanceManager.java:1093)
E/AndroidRuntime(31601): at com.facebook.react.ReactInstanceManager.processPackages(ReactInstanceManager.java:1064)
E/AndroidRuntime(31601): at com.facebook.react.ReactInstanceManager.createReactContext(ReactInstanceManager.java:996)
E/AndroidRuntime(31601): at com.facebook.react.ReactInstanceManager.access$600(ReactInstanceManager.java:107)
E/AndroidRuntime(31601): at com.facebook.react.ReactInstanceManager$4.run(ReactInstanceManager.java:832)
E/AndroidRuntime(31601): at java.lang.Thread.run(Thread.java:818)
W/ActivityManager(  551):   Force finishing activity 1 com.virosample/.MainActivity
```

we also tried downgrading to `2.4.0` but saw the `dlopen` call fail with `libarcore_sdk.so` in a similar way to #192

Thanks!

**Comment by radvani** [comment]
Hi @Thomas101, it looks like 5.1.1 devices are having issues linking in the STL library. We're investigating a fix for this now. Thanks for the bug report.

**Comment by radvani** [comment]
This has been resolved and the fix will be available in our next release, targeted for mid to late May. Thanks for your patience!

**Comment by VikAdvani** [comment]
Hi @Thomas101, this has been fixed in our latest major release v2.7 -> See release notes and upgrade instructions here -> https://docs.viromedia.com/docs/releases
